code/__HELPERS/lists.dm

```diff
@@ -57,11 +57,17 @@
 	return 0
 
 //Checks for specific types in a list
-/proc/is_type_in_list(var/atom/A, var/list/L)
+/proc/is_type_in_list(atom/A, list/L)
```

Comment:
Ah, but it will get people to stop complaining at you more quickly than arguing with them will